src/Test/Runner.elm

```diff
@@ -147,3 +156,46 @@ formatLabels formatDescription formatTest labels =
                 |> List.map formatDescription
                 |> (::) (formatTest test)
                 |> List.reverse
+
+
+{-| A `Shrinkable a` is an opaque type that allows you to obtain a value of type
+`a` that is smaller than the one you've previously obtained.
+-}
+type Shrinkable a
+    = S { down : LazyList (RoseTree a), over : LazyList (RoseTree a) }
```

Comment:
I still like passing named arguments explicitly, that's why I'm using the record in the first place, but I got rid of `S` as a name.